Tests/mpc-kit-swiftTests/mpc_kit_swiftTests.swift

```diff
@@ -120,7 +120,8 @@ final class mpc_kit_swiftTests: XCTestCase {
 
         let keyDetails = try await coreKitInstance.loginWithJwt(verifier: verifier, verifierId: email, idToken: token)
         
-        let hash = Data(hex: "010203040506").sha256()
+        let hash =  try Data(hex: "010203040506").sha3(varient: Variants.KECCAK256)
```

Comment:
Note that curvelib also has keccak available.

```try keccak256(data: data.data(using: .utf8)!)```

Reply:
done